lib/datadog/appsec/contrib/devise/tracking.rb

```diff
@@ -13,24 +13,32 @@ module Tracking
           SIGNUP_EVENT = 'users.signup'
 
           def self.track_login_success(trace, span, user_id:, **others)
+            return if trace.nil? && span.nil?
```

Comment:
Does it make sense to use `||` instead of `&&` ?

Reply:
Oh, right, I reversed things: both `trace` *and* `span` must be non-nil to continue, so if either is nil we must return.

Reply:
Fixed in [6887455](https://github.com/DataDog/dd-trace-rb/pull/3645/commits/6887455ac35bfa0ef1539ace1c6c7638a8418d4f)